stories/utils/generate-widget-stories.js

```diff
@@ -36,12 +36,12 @@ const { components: { Widget } } = Widgets;
  *
  * @since 1.16.0
  *
- * @param {string} moduleSlug Module slug.
- * @param {string|null} url Current entity URL.
- * @param {Function} cb Callback for additional setup.
+ * @param {(string|Array)} moduleSlugs Module slug or slugs to activate.
+ * @param {string|null}   url          Current entity URL.
+ * @param {Function}     cb            Callback for additional setup.
  * @return {Function} A function to set up registry for widget stories.
  */
-function getSetupRegistry( moduleSlug, url, cb = () => {} ) {
+function getSetupRegistry( moduleSlugs, url, cb = () => {} ) {
```

Comment:
I'd prefer to still keep this called `moduleSlug` just because that's the usual use-case (a single one).